scipy/stats/_continuous_distns.py

```diff
@@ -9062,6 +9062,185 @@ def _updated_ctor_param(self):
         return dct
 
 
+class studentized_range_gen(rv_continuous):
+    """
+    The studentized range continuous random variable.
+
+    %(before_notes)s
+
+    See Also
+    --------
+    t: Student's t distribution
+
+    Notes
+    -----
```

Comment:
Please add the PDF like other distributions.  See e.g. https://github.com/scipy/scipy/pull/12694/files.

Reply:
I don't see this change.

Reply:
Thumbs up was to acknowledge, it was being difficult so I am going to circle back after focusing on the CI.